### PR TITLE
chore(demo): demo add copy code button

### PR DESCRIPTION
**Pull request by ahwgs**

### CODE REVIEW

File: demo/utils/CopyCodeButton.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <template #icon>
+      <n-icon>
+        <copy-icon />
+      </n-icon>
+    </template>
+  </n-button>
+</template>
+
+<script>
+import CopyIcon from '@vicons/fluent/Copy24Regular.js'
```
Comment (07akioni) [inline demo/utils/CopyCodeButton.vue:18]: Use Copy16 instead. It's better for small icon.

File: demo/utils/CopyCodeButton.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import copy from 'copy-text-to-clipboard'
```
Comment (07akioni) [inline demo/utils/CopyCodeButton.vue:19]: Can you change it to a native implementation? I think we don't need to add a dep for this.

File: demo/utils/ComponentDemo.vue [hunk 3]
```diff
@@ -76,12 +88,13 @@ import { useDisplayMode } from '../store'
 import { i18n } from '../utils/composables'
 import EditOnGithubButton from './EditOnGithubButton.vue'
 import editInCodeSandboxButton from './EditInCodeSandboxButton.vue'
-
+import CopyCodeButton from './CopyCodeButton.vue'
 export default {
```
Comment (07akioni) [inline demo/utils/ComponentDemo.vue:92]: Add an empty line here.

Comment (ahwgs) [reply demo/utils/ComponentDemo.vue:92]: No blank line

File: demo/utils/CopyCodeButton.vue (hunk 2 above)
Comment (ahwgs) [reply demo/utils/CopyCodeButton.vue:19]: ok

File: demo/utils/CopyCodeButton.vue (hunk 1 above)
Comment (ahwgs) [reply demo/utils/CopyCodeButton.vue:18]: image

16 looks smaller than the others

File: demo/utils/ComponentDemo.vue [hunk 4]
```diff
@@ -130,13 +143,17 @@ export default {
...
+          copySuccess: '复制成功'
         },
         'en-US': {
           show: 'Show Code',
           hide: 'Hide Code',
           editOnGithub: 'Edit on Github',
-          editInCodeSandbox: 'Edit in CodeSandbox'
+          editInCodeSandbox: 'Edit in CodeSandbox',
+          copyCode: 'Copy Code',
+          copySuccess: 'Copy Success'
```
Comment (07akioni) [inline demo/utils/ComponentDemo.vue:156]:
```suggestion
          copySuccess: 'Successfully Copied'
```

**Pull request merged**